be/src/formats/parquet/predicate_filter_evaluator.h

```diff
@@ -46,8 +46,16 @@ class PredicateFilterEvaluatorUtils {
 };
 
 struct PredicateFilterEvaluator {
+    enum Evaluator {
+        NONE = 0,
+        STATS_ZONEMAP = 1,
+        PAGE_INDEX = 2,
```

Comment:
why separate `PAGE_INDEX` from `STATS_ZONEMAP`?

Reply:
maybe ROWGROUP_ZONEMAP and PAGE_INDEX_ZONEMAP